packages/invoices/src/api.ts

```diff
@@ -148,15 +181,23 @@ export const listQueryFn = async <T extends ObjectWithStringKeys>(queryParams: {
   first: number;
   offset: number;
   sortBy: SortBy<T>;
-}): Promise<{ data: Invoice[]; totalLength: number }> => {
-  const { first, offset, sortBy } = queryParams;
-
-  const { invoices } = await request(Environment.API_URL, getListQuery(), {
-    first,
-    offset,
-    sort: sortBy.key,
-    desc: sortBy.isDesc,
-  });
+}): Promise<{ nodes: Invoice[]; totalCount: number }> => {
+  const {
+    first = 20,
+    offset = 0,
+    sortBy = { key: 'TYPE', isDesc: false },
+  } = queryParams;
+
+  const { invoices } = await request(
+    'http://localhost:8000/graphql',
```

Comment:
I haven't updated the mock server so this query is targeting this url which is the default for the rust server, for now. Lots of cross over between things ..